templates/login.html

```diff
@@ -2,18 +2,60 @@
 
 {% block links %}
 <link rel="shortcut icon" href="{{url_for('static', filename='booksearch.jpg')}}" />
-<link rel ="stylesheet" href = "{{url_for('static', filename='reg.css')}}" />
+<link rel ="stylesheet" href = "{{url_for('static', filename='book_page.css')}}" />
 {% endblock %}
 {% block styles %}
-<style>
-  body {
-background-image: url('static/books.jpg'); 
-color: white; 
-background-size: cover; 
-background-repeat: no-repeat;
-}
+<style> 
 </style>
 {% endblock %}
+{% block scripts%}
+  <script type="text/javascript"> 
+  function bookdisplay(ele) {
+    console.log(ele.id)
+    var requestobj = {}
+    requestobj["isbn"] = ele.id
+    var url = "/api/book?isbn="+ele.id
+    // document.getElementById("id").innerHTML=requestobj.isbn;
+    var xhr  = new XMLHttpRequest()
+    xhr.open('GET', url)
+    xhr.onload = function () {
+	  var data = JSON.parse(xhr.responseText);
+    // console.log(users);
+	  if (xhr.status == "200") {
+		  // console.log("Success");
+      
+            var htmltext = "<br><h1 style = 'text-align:center'>The Book Details Here!!!</h1><br><br> <div class = 'container' style = 'color:white' align = 'center'>"+
+            "<div class = 'container' style = 'background-color: bisque'><div class = 'row' ><div class = 'col-lg-4'><img  style = 'margin-left:250px;' src = 'http://covers.openlibrary.org/b/isbn/"+data.isbn+".jpg' alt = "+data.title+"/></div>"+
+            "<div class = 'col-lg-4'><table><tr><th>ISBN:</th><td><b>"+data.isbn+"</b></td></tr><tr><th>TITLE:</th>"+
+            "<td><b>"+data.title+"</b></td></tr><tr><th>AUTHOR:</th>"+
+            "<td><b>"+data.author+"</b></td></tr><tr><th>YEAR:</th>"+
+              "<td><b>"+data.year+"</b></td></tr></table></div><div class = 'col-lg-4'></div></div></div><br><br><br><br><br><br><br><br><br><h2 style = 'text-align: center;'> Previous Comments</h2>";
+              
+
+            for(var i = 0; i < data.reviews.length; i++) {
+              
+                htmltext= htmltext + "<div><div class='card ' style='width: 40rem; background-color: bisque;'>"+
+                        "<div class='card-body'><h5 class='card-title' style = 'color:black'>Email:"+data.reviews[i]["email"]+"</h5><h6 class='card-subtitle mb-2 text-muted' style = 'color:black'>Rating: "+data.reviews[i]["rating"]+
+                        "</h6><p class='card-text' style = 'color:black'>Comment: "+data.reviews[i]["comment"]+"</p></div></div></div>"  
+            }
+            htmltext = htmltext 
+            document.getElementById("reviewresults").innerHTML=htmltext;
+
+	  } else {
+		  // console.error("Fail");
+      if (xhr.status == 404) {
```

Comment:
Use the error message from the API itself.

Reply:
updated, bro..